src/greyVest/GridItem.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+let GridItem = ({
```

Comment:
I'm not thrilled about the start,end,span props. I personally like to drop down to css for this with `row` and `column` as you have it here. It encourages actually learning grid

Reply:
You got me - I just didn't want to actually learn grid myself 😂

Yeah, I don't know what I was thinking with `span`. I'm still inclined toward keeping `start` and `end` though, even if they're not strictly necessary, since they directly mirror a commonly-used part of the CSS API.

Reply:
Since we are copying the API of another component, we should include a link referencing it. A comment is sufficient

Reply:
Actually we just got inspired by it. After going over it, we didn't copy most of their API